gradle.properties

```diff
@@ -1,24 +1,5 @@
-# Project-wide Gradle stream_settings.
-# IDE (e.g. Android Studio) users:
-# Gradle stream_settings configured through the IDE *will override*
-# any stream_settings specified in this file.
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
-# Specifies the JVM arguments used for the daemon process.
-# The setting is particularly useful for tweaking memory stream_settings.
```

Comment:
Fun fact: we changed every mention of the word `settings` in the project to `stream_settings` a while back, while refactoring a string resource's name 😄